src/webgpu/shader/execution/expression/expression.ts

```diff
@@ -379,8 +379,8 @@ function packScalarsToVector(
 }
 
 /** @returns a set of flushed and non-flushed floating point results for a given number. */
-function calculateFlushedResults(value: number): Set<Scalar> {
-  return new Set([f64(value), f64(flushSubnormalNumber(value))]);
```

Comment:
Would it make more sense to filter out the duplicate entries here? so that they don't propagate up and cause unneeded comparisons elsewhere.
Something like 
```
const subnormal_value = flushSubnormaNumber(value);
if (subnormal_value === value) {
  return  [f64(value)];
} else {
  return  [f64(value), f64(subnormal_value)];
}
```

Reply:
We could add this as an optimization, but it's not sufficient on its own because we can add more values via anyOf().